scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncher.java

```diff
@@ -148,12 +144,18 @@ public void doTask(ExecutableContainer executableContainer, TaskResult[] previou
             progressFileReader.start(dataspaces.getScratchFolder(), taskId);
 
             TaskContext context = new TaskContext(executableContainer, initializer, previousTasksResults,
-                dataspaces.getScratchURI(), dataspaces.getInputURI(), dataspaces.getOutputURI(),
-                dataspaces.getUserURI(), dataspaces.getGlobalURI(), progressFileReader.getProgressFile()
-                        .toString(), getHostname());
+                    dataspaces.getScratchURI(), dataspaces.getInputURI(), dataspaces.getOutputURI(),
+                    dataspaces.getUserURI(), dataspaces.getGlobalURI(), progressFileReader.getProgressFile()
+                    .toString(), getHostname());
 
             File workingDir = getTaskWorkingDir(context, dataspaces);
 
+            logger.info("Task working dir : " + workingDir);
```

Comment:
@lpellegr  Good point, in the case we see it can take more than 3 days we can split the stories.
@fviale looks like those logs are needed at INFO level so :)